src/monattack.cpp

```diff
@@ -5283,6 +5284,41 @@ bool mattack::stretch_attack( monster *z )
     return true;
 }
 
+bool mattack::zombie_fuse( monster *z )
+{
+    const mfaction_id zombo = mfaction_id( "zombie" );
+    const tripoint zpos = z->pos();
+    Creature *critter = nullptr;
+    for( int x = zpos.x - 1; x < zpos.x + 2; x++ ) {
```

Comment:
```suggestion
    for( const tripoint &p : g->m.points_in_radius( z->pos(), 1 ) ) {
```
That gives you all 9 points at and around `z->pos()` in one loop and as a single `tripoint`.